README.md

```diff
@@ -1 +1,80 @@
-# Capstone-project
\ No newline at end of file
+# Capstone-project
```

Comment:
- [OPTIONAL] Kindly delete this title because line number 6 has a title. let make the line number 6 be the first line following by the description

Reply:
done